tests/libvmi/vmi.go

```diff
@@ -135,3 +135,13 @@ func baseVmi(name string) *v1.VirtualMachineInstance {
 	}
 	return vmi
 }
+
+// WithResourceCpu specifies the vmi cpu resource.
+func WithResourceCPU(value string) Option {
```

Comment:
for the benefit of future git-loggers, please state why adding this function is a good idea (e.g we have multiple tests needing something like it).

nit: it feels better if placed next to `WithResourceMemory`.

Reply:
Done